ivy/data_classes/container/experimental/manipulation.py

```diff
@@ -3678,3 +3678,57 @@ def soft_thresholding(
             thresholded tensor on which the operator has been applied
         """
         return self.static_soft_thresholding(self, threshold, out=out)
+
+    @staticmethod
+    def _static_put_along_axis(
+        arr: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+        indices: Union[ivy.Array, ivy.NativeArray, ivy.Container],
```

Comment:
We are supposed to add the `key_chains`, `map_sequences` etc. arguments when defining the container methods.